library/psa_crypto.c

```diff
@@ -3364,7 +3368,426 @@ psa_status_t psa_verify_hash_abort(
     return( PSA_SUCCESS );
 }
 
+/****************************************************************/
+/* Asymmetric interruptible cryptography internal               */
+/* implementations                                              */
+/****************************************************************/
+
+static uint32_t mbedtls_psa_interruptible_max_ops =
+                                            PSA_INTERRUPTIBLE_MAX_OPS_UNLIMITED;
+
+void mbedtls_psa_interruptible_set_max_ops( uint32_t max_ops )
+{
+#if (defined(MBEDTLS_PSA_BUILTIN_ALG_ECDSA) || \
```

Comment:
Would this be a PSA_WANT kind of define? I may need some guidance on their construction if so.

Reply:
I think at some point we need some official `PSA_WANT` thing about restartable operations, but that might require a bit more thinking. In the short term I was thinking something `MBEDTLS_PSA_BUILTIN_INTERRUPTIBLE_ECDSA` and preferably not public, so that we're free to change it in the future.

Reply:
Having discussed this with Gilles, he feels we should probably move this to a later commit, as he has yet to decide the level of granularity he wants for either the `MBEDTLS_PSA_BUILTIN_` defines or the `PSA_WANT_` defines. This will be something done straight after this PR though.

Reply:
This is tracked as #7029 now, so I'm marking this conversation as resolved.